Makefile

```diff
@@ -11,9 +11,11 @@ virtualenv:
 
 requirements: virtualenv requirements.txt
 	${VIRTUALENV_ROOT}/bin/pip install -r requirements.txt
+	npm install
```

Comment:
This should probably be a separate make step that `frontend_build` could depend on